libparsec/crates/protocol/schema/anonymous_account_cmds/account_create_proceed.json5

```diff
@@ -7,43 +7,12 @@
     "req": {
       "fields": [
         {
-          // Token received by email following use of `account_create_send_validation_email`
-          "name": "validation_token",
-          "type": "EmailValidationToken"
+          "name": "email",
```

Comment:
`email` field should also be moved in the step:
- as `email: EmailAddress` for `Number0CheckCode`
- as `human_handle: HumanHandle` for `Number1Create`

This is because in `Number1Create` we currently also provide the `human_label` field, but as a string so without any guarantee regarding its content.
But `HumanHandle` constructor does some validation on the label (e.g. a label cannot contain a `<` character), and we don't have a dedicated type that represent a validated human label.